core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -261,7 +261,7 @@ namespace kagome::blockchain {
     }
     auto start_block_number = block_number_res.value();
 
-    primitives::BlockNumber finish_block_number;
+    primitives::BlockNumber finish_block_number; // NOLINT
```

Comment:
initialization is more appropriate than hiding the problem.

Reply:
not critical, either way is fine